typescript/cli/src/tests/warp/warp-rebalancer.e2e-test.ts

```
@@ -0,0 +1,1621 @@
+          '✅ Rebalance successful',
+          `{ context: 'WeightedStrategy', numberOfRoutes: 0 } Found rebalancing routes`,
+          'No routes to execute',
+        ],
+        { timeout: 30000, checkFrequency: 1000 },
+      );
+    } finally {
+      await relayer.kill();
+    }
+  });
+
+  describe.only('manual rebalance', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)